lib/iris/analysis/__init__.py

```diff
@@ -1597,7 +1601,13 @@ def regridder(self, src_grid, target_grid):
                 `callable(cube)`
 
             where `cube` is a cube with the same grid as `src_grid`
-            which is to be regridded to the `target_grid`.
+            that is to be regridded to the `target_grid`.
+            Thus for the callable returned by
+            `Linear().regridder(src_grid, target_grid)`, cube must
+            be on the same grid as src_grid.
+
+            This callable would typically be used by
+            :class:`iris.cube.Cube.regrid`.
```

Comment:
> I'd second highlighting the above use pattern in the doc-string 

:+1: but actually I think I'm going to replace it with a link to a section in the user guide chapter on interpolate and regrid. This section (that doesn't currently exist!) will include example code very similar to the above but will be a little longer and give more context.

Doing this will prevent duplicating this precise example in every `regridder` docstring for each regridding scheme :scream_cat: